generators/server/templates/dotnetcore/test/Project.Test/Controllers/ProfileInfoControllerIntTest.cs.ejs

```diff
@@ -0,0 +1,46 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+using FluentAssertions;
+using MyCompany.Test.Setup;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace MyCompany.Test.Controllers {
+    public class ProfileInfoControllerIntTest
+    {
+
+        private readonly NhipsterWebApplicationFactory<TestStartup> _factory;
+
+        public ProfileInfoControllerIntTest()
+        {
+            _factory = new NhipsterWebApplicationFactory<TestStartup>();
+        }
+
+        [Fact]
+        public async Task TestGetProfileInfos()
+        {
+            var client = _factory.CreateClient();
+            var response = await client.GetAsync("/management/info");
+            response.StatusCode.Should().Be(HttpStatusCode.OK);
+
+            var json = JToken.Parse(await response.Content.ReadAsStringAsync());
+            json.SelectToken("$.display-ribbon-on-profiles").Value<string>().Should().Be("dev");
+            // json.SelectToken("$.activeProfiles").ToObject<IEnumerable<string>>().Should().Contain(new[] { "swagger", "prod" });
```

Comment:
The variable ASPNETCORE_ENVIRONMENT is for this line can you uncomment ?

Reply:
> The variable ASPNETCORE_ENVIRONMENT is for this line can you uncomment ?

Yes, I forgot it !